torchaudio/metrics.py

```diff
@@ -1,9 +1,17 @@
 from typing import List, Union
 
 
-def levenshtein_distance(r: Union[str, List[str]], h: Union[str, List[str]]):
+def levenshtein_distance(r: Union[str, List[str]], h: Union[str, List[str]]) -> int:
```

Comment:
From a minimalism perspective, I can see that List[str] arguments can be useful, but aside from syntactic sugar, what else do they offer users?

Reply:
Also, can you mix List[str] and str?

Reply:
The function runs with any two arbitrary sequences. I updated this. That being said, a user would only really using this for two Sequence[T] for the same T.

Reply:
relate to [comment](https://github.com/pytorch/audio/pull/927#discussion_r497641380).